src/Sentry.Extensions.Logging/Sentry.Extensions.Logging.csproj

```diff
@@ -7,6 +7,12 @@
     <AssemblyName>Sentry.Extensions.Logging</AssemblyName>
     <RootNamespace>Sentry.Extensions.Logging</RootNamespace>
     <Description>Official Microsoft.Extensions.Logging integration for Sentry - Open-source error tracking that helps developers monitor and fix crashes in real time.</Description>
+    <Nullable>enable</Nullable>
```

Comment:
If this is the last package, can we move this to the root Directory.Build.props?
Or actually just the `src`